docker-compose.yml

```
@@ -327,7 +327,7 @@ services:
   import:
     image: georchestra/datafeeder-frontend:latest
     healthcheck:
-      test: ["CMD-SHELL", "wget --no-verbose --tries=1 --spider http://localhost:80/ >/dev/null || exit 1"]
+      test: ["CMD-SHELL", "curl -s -f http://localhost:80/ >/dev/null || exit 1"]
```

there is no need to use curl instead. why this does not work with wget is that datafeeder do not listen on ipv6 localhost.
and wget do not do https://en.wikipedia.org/wiki/Happy_Eyeballs

so it fails. but you can easily fix that this way:
```suggestion
      test: ["CMD-SHELL", "wget --no-verbose --tries=1 --spider http://127.0.0.1:80/ >/dev/null || exit 1"]
```

but IMO this should be fixed inside the datafeeder container for the reason why it do not listen on ipv6.